examples/resources/security_policy/resource_cluster_group_security_policy.tf

```diff
@@ -0,0 +1,84 @@
+# Uncomment the commented blocks to customize
+# Create Tanzu Mission Control securitypolicy entry
+resource "tanzu-mission-control_security_policy" "create_cluster_group_security_policy" {
+  name = "tf-sp-test"
+
+  scope {
+#    cluster {
```

Comment:
can you add multiple examples each covering different usecases instead of commenting them out

Reply:
Sure

Reply:
Done